app/controllers/application_controller.rb

```diff
@@ -8,9 +8,15 @@ class ApplicationController < ActionController::Base
 
   add_flash_types :success, :info, :warning, :danger
   before_action :authenticate_user!
+  around_action :set_current_user
   around_action :use_logidze_responsible, only: %i[create destroy update transfer] # rubocop:disable Rails/LexicallyScopedActionFilter
   around_action :switch_locale
 
+  def set_current_user
+    Current.user = current_user
+    yield
```

Comment:
**nitpick:** Don't think this should be here

Reply:
It is necessary since this method is called via an `around_action` callback. Without the `yield` the UI no longer loads and you just get blank pages.

Reply:
Ok, that sounds good. Thaks for clarifying